src/language/visitor.js

```diff
@@ -64,7 +64,7 @@ export const QueryDocumentKeys = {
     'directives',
     'selectionSet',
   ],
-  VariableDefinition: ['variable', 'type', 'defaultValue'],
+  VariableDefinition: ['variable', 'type', 'defaultValue', 'directives'],
```

Comment:
These typically follow the order found in the grammar, but this seems inconsistent

Reply:
My mistake. Consistent for this PR but inconsistent with the spec RFC